handler/core/docs/asciidoc/dataobjects.adoc

```diff
@@ -8,6 +8,7 @@
 [frame="topbot"]
 |===
 ^|Name | Type ^| Description
+|[[actionLogMode]]`@actionLogMode`|`link:enums.html#ActionLogMode[ActionLogMode]`|-
```

Comment:
It shouldn't be possible to modify log mode at the Action level.

Reply:
true. I haven't updated adoc. thanks!